### PR TITLE
better messeges and defaults for cfn

### PR DESCRIPTION
**Pull request by iakov-aws**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cfn-templates/cid-cfn.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ Metadata:
           - CURBucketPath
           - CURIsManagedByCloudFormation
       - Label:
-          default: Available dashboards. Keep all 'no' for the first deployment. Go to QuickSight, change Permission and after that update the stack putting to 'yes' 3 first dashboards.
+          default: Available dashboards. CUDOS CostIntelligenceDashboard and KPI are CUR based, others requre installation of an additional Lab.
```
Comment (yprikhodko) [inline cfn-templates/cid-cfn.yml:19]: I'd suggest to re-phrase to 'Trusted Advisor and Compute Optimiser Dashboards require deployment of Optimization Data Collection Lab (https://wellarchitectedlabs.com/cost/300_labs/300_optimization_data_collection/)

File: cfn-templates/cid-cfn.yml [hunk 2]
```diff
@@ -25,7 +25,7 @@ Metadata:
           - DeployComputeOptimizerDashboard
           - CostOptimizationDataCollectionBucketPath
       - Label:
-          default: 'Optional Parameters (do not change unless you know why)'
+          default: 'Optional Parameters. DO NOT CHANGE!'
```
Comment (yprikhodko) [inline cfn-templates/cid-cfn.yml:28]: DO NOT CHANGE! looks unfriendly, do we really need to change this?

File: cfn-templates/cid-cfn.yml [hunk 3]
```diff
@@ -108,26 +108,26 @@ Parameters:
...
   DeployKPIDashboard:
     Type: String
     Description: Deploy KPI Dashboard
-    Default: "no"
+    Default: "yes"
     AllowedValues: ["yes", "no"]
   DeployTAODashboard:
     Type: String
-    Description: Deploy Trusted Advisor Organisational Dashboard (TAO) - You need Optimisation Data Collection Lab to be installed
+    Description: ADDITIONAL - Deploy Trusted Advisor Organisational Dashboard (TAO) - WARNING! You need Optimisation Data Collection Lab to be installed https://wellarchitectedlabs.com/cost/300_labs/300_optimization_data_collection/
```
Comment (yprikhodko) [inline cfn-templates/cid-cfn.yml:125]: Before deploying this dashboard, you need Optimisation Data Collection Lab to be installed first https://wellarchitectedlabs.com/cost/300_labs/300_optimization_data_collection/

**Review by yprikhodko — CHANGES_REQUESTED** [review]

File: cfn-templates/cid-cfn.yml (hunk 2 above)
Comment (iakov-aws) [reply cfn-templates/cid-cfn.yml:28]: Yes. These parameters are not very well tested and error messages can be really confusing. Removing them will require more work, so it is better to put a quick warning until we do so

**Pull request closed without merging**